torch/fx/experimental/proxy_tensor.py

```diff
@@ -444,7 +446,7 @@ def getattr(self, attr, attr_val, parameter_proxy_cache):
         return attr_val
 
     def create_arg(self, a: Any):
-        if isinstance(a, torch.nn.Parameter):
+        if isinstance(a, (torch.nn.Parameter, torch._subclasses.FakeTensor)):
```

Comment:
This I think is unnecessary, a properly fakeified parameter should still isinstance(x, Parameter)